File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.swift
import Foundation
let N = Int(readLine()!)!
    let sizes = readLine()!.split { $0 == " " }.map { Int($0)! }
    let TP = readLine()!.split { $0 == " " }.map { Int($0)! }
    let (T, P) = (TP[0], TP[1])
    
    print(sizes.map { Int(ceil(Double($0) / Double(T))) }.reduce(0, +))
    print(N / P , N % P)

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.py
import sys
import math

readLine = sys.stdin.readline

N = int(readLine())
sizes = list(map(int, readLine().split()))
T, P = map(int, readLine().split())
F = 0
for s in sizes:
    F += math.ceil(s / T)
print(F)
print(N // P, N % P)